stringify.js

```diff
@@ -1,11 +1,17 @@
 const escapeString = (str) => str && str.replace(/[\n\r]/g, '')
-const stringify = (data, depth = 5) => {
+// Won't work for minified class names however we need something fast
+// That does not require iterating through everything
+const fetchTypeName = (data) => data && data.constructor && data.constructor.name
```

Comment:
you can also have multiple different classes with the same name. but I guess we can disambiguate within the formatter function

Reply:
Indeed, I wanted to use `instanceOf` however that means running through the entire list of formatters instead of a constant lookup, although if we know the list is small, it can be done